config/internal/mariadb/pvc.yaml.tmpl

```diff
@@ -10,7 +10,7 @@ spec:
   accessModes:
     - ReadWriteOnce
   {{- if .MariaDB.StorageClassName }}
```

Comment:
not part of this review but the extraneous `-` at the beginning here feels like it should break something?

Reply:
nvm, this is part of trimming in templates.  I'm not entirely sure it's necessary here but not worth blocking the review over